docs/dxp/7.x/en/site-building/creating-pages/building-and-managing-content-pages/using-widgets-on-a-content-page.md

```diff
@@ -0,0 +1,52 @@
+# Using Widgets on Content Pages
+
+The Widgets section functions just like the [*Add* menu on a Widget Page](../using-widget-pages/adding-widgets-to-a-page.md). The full list of available widgets is displayed, and you can add them to the page. The main difference is that only the main configuration options for Widgets are available on Content Pages.
+
+## Adding Widgets on Content Pages 
+
+Follow these steps to add a widget to a Content Page:
+
+1. While [building the Content Page](./building-content-pages.md), open the [Widgets section](./content-pages-overview.md#widgets) and either browse through the categories of available widgets until you find the one you want, or you can search for widgets by name.
+1. Drag the widget to the column and row of the layout where you want to place it.
```

Comment:
@mwilliams2014, this might be confusing for the user, since we also have sections

Reply:
Good point. I can use a different word. Perhaps "Panel" or "Menu".